unit-test/background/classes/https-redirects.es6.js

```diff
@@ -3,110 +3,143 @@ const tk = require('timekeeper')
 const pixel = require('../../../shared/js/background/pixel.es6')
 let httpsRedirects
 
+/**
+ * HELPERS
+ */
+
 const setup = () => {
     httpsRedirects = new HttpsRedirects()
     spyOn(pixel, 'fire')
 }
 
+let id = 5
+
 const teardown = () => {
     httpsRedirects.resetMainFrameRedirect()
     tk.reset()
+
+    id = 5
 }
 
 const fastForward = (ms) => {
     tk.travel(new Date(Date.now() + ms))
 }
 
+const getMainFrameRequest = () => {
+    id += 1
+
+    return {
+        requestId: id,
+        type: 'main_frame',
+        url: 'http://example.com'
+    }
+}
+
+const getImageRequest = () => {
+    id += 1
+
+    return {
+        requestId: id,
+        type: 'image',
+        url: 'http://example.com/cat.gif'
+    }
+}
```

Comment:
Took a moment to simplify the tests by adding helpers. We were creating these objects all over the place unnecessarily. Hopefully the tests are much more readable now.

Reply:
Thanks for doing this! Good idea.